docs/spark/toc.yml

```diff
@@ -3,6 +3,8 @@
 - name: Overview
   expanded: true
   items:
+  - name: What is Apache Spark?
+    href: what-is-spark.md
```

Comment:
nit: typically if you run the SEO tests on CATS, they will flag when you have duplicated words on the path.